rioxarray/_io.py

```diff
@@ -532,6 +532,15 @@ def _get_rasterio_attrs(riods):
             attrs["units"] = riods.units[0]
         else:
             attrs["units"] = riods.units
+    if hasattr(riods, "colormap"):
+        # A dict containing the colormap for a band
+        try:
+            attrs["colormap"] = riods.colormap(1)
```

Comment:
This only gets the colormap for the first band. I imagine it would be good to handle this for all of the bands. [docs](https://rasterio.readthedocs.io/en/latest/topics/color.html#reading-colormaps)

Reply:
Yep, probably need to loop through all the bands, but need to think about https://github.com/corteva/rioxarray/pull/414/files#r722041137 first :slightly_smiling_face: